### PR TITLE
fixed can`t compile when using jdk7 or jdk8

### PR DESCRIPTION
**Pull request by yuxuanchiadm**
Must set bootclasspath when doing cross version compile. Otherwise you will get a warning. then -Werror will let warning be error.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -81,6 +81,10 @@ dependencies {
 }
 
 compileJava {
+  if(System.env.JDK6_HOME != null) {
+    options.fork = true
+    options.bootClasspath = "$System.env.JDK6_HOME/jre/lib/rt.jar"
```
Comment (cazzar) [inline build.gradle]: This implies everyone has set up a JDK6_HOME variable

**Comment by AbrarSyed** [comment]
no.

either you comment out that section if you wanan built it yourself, or you set up your environment properly with this: https://gist.github.com/AbrarSyed/9440192

**Comment by LexManos** [comment]
You could scan for JDK installs by finding the default and popping a few directories.

**Comment by matthewprenger** [comment]
I'd just remove [this](https://github.com/MinecraftForge/ForgeGradle/blob/FG_1.2/build.gradle#L85) line. Failing the build because of a warning is mildly annoying.

**Comment by AbrarSyed** [comment]
I like that line, it keeps my code free of warnings. I would do the unchecked thing too, but apparently
it doesnt respect the annotations to override it..

but anyways, my earlier point still stands. Either comment it out, or setup your bootstrap classpaths.

**Pull request closed without merging**